hs_core/views/utils.py

```diff
@@ -967,9 +983,18 @@ def unzip_file(user, res_id, zip_with_rel_path, bool_remove_original, overwrite=
             for folder in listfolders(istorage, unzip_path):
                 destination_folder = os.path.join(working_dir, folder)
                 destination_folders.append(destination_folder)
+
+            irods_files = []
+            for unzipped_file in unzipped_files:
+                irods_files.append(IrodsFile(unzipped_file, istorage))
+            from hs_file_types.utils import identify_metadata_files
```

Comment:
Again, it'd be more structured to move the import to the beginning.

Reply:
moved